pillar/edx/residential_ansible_vars.sls

```diff
@@ -24,10 +21,6 @@
 {% if cache_configs is mapping %}
   {% set cache_configs = [cache_configs] %}
 {% endif %}
-{% set xqueue_mysql_creds = salt.vault.read(
-    'mysql-{env}/creds/xqueue-{purpose}'.format(
-        env=environment,
-        purpose=purpose)) %}
 {% set xqwatcher_xqueue_creds = salt.vault.read(
```

Comment:
Missed?

Reply:
This was left due to the fact that part of the return value is referenced as a key in the nested data structure.